src/converters/tablecell-post-fixer.js

```diff
@@ -126,6 +126,10 @@ function getElementsToCheck( view ) {
 // - span: for single paragraph with no attributes.
 // - p   : in other cases.
 function ensureProperElementName( currentViewElement, mapper, writer ) {
+	if ( !currentViewElement.root.is( 'rootElement' ) ) {
```

Comment:
Could you leave here a comment about how root can not be `rootElement` and why `false` should be returned in such case?

Reply:
Done.